ts/features/payments/home/utils/useTransactionHistory.ts

```diff
@@ -0,0 +1,23 @@
+import { fetchTransactionsRequestWithExpBackoff } from "../../../../store/actions/wallet/transactions";
```

Comment:
This is a hook, an `hooks` folder could be a better fit. What do you think?

Reply:
To be honest, between `utils` and `hooks` I don't see that big of a difference, especially when there is a single hook in the whole context, but why not?

Reply:
nonetheless, addressed in [1093572](https://github.com/pagopa/io-app/pull/5535/commits/109357292f915296356d52db699dbdf8f128b63f) 👍